### DISCUSSION TITLE
`EXTERNAL WALLET` -> `Metamask` not working properly

### DISCUSSION THREAD

**Discussion by MizukiSonoko** [Web3Auth/Web3Auth#607] [Help]
Hello, currently I develop web service using Web3Auth.
And I have faced this problem.

In deployed site (Whitelisted).
```js
web3auth = new Web3Auth({
  clientId, // Client ID 
  chainConfig: {
    chainNamespace: CHAIN_NAMESPACES.EIP155,
    chainId: "0x1",
    rpcTarget: $config.web3auth.rpcTarget, // From Infura.
  },
});
await web3auth.initModal();

provider = await web3auth.connect();
```
Open modal and,
`EXTERNAL WALLET` -> `Metamask`.
I expect open Metamask modal, but it not working properly.
<img width="366" alt="image" src="https://user-images.githubusercontent.com/6281583/189924246-eded39f5-e34a-42fe-95c7-2dd7b84e506d.png"> 
So What's I should do? (As a side note. In `localhost:3000` , it works.  

Regards

**Comment by rinzler99** [comment]
If you take look at the demo app (https://demo-app.web3auth.io/) this works fine. Please share console logs if any.

**Reply by rinzler99** [reply]
Try to go through the example code of the demo app.

**Reply by MizukiSonoko** [reply]
Thanks!  yes your site ( https://demo-app.web3auth.io/ ) works fine. but not so on my page.
And there is no message on console....
Coinbase wallet, Wallet Connect, and Torus Wallet works fine. only Metamask dosen't works...
Metamask modal not displayed.

**Reply by MizukiSonoko** [reply]
This code from dashboard usage, but MetaMask not works...
```
import { CHAIN_NAMESPACES } from "@web3auth/base";
import { Web3Auth } from "@web3auth/web3auth";
import { onMounted } from "vue";

const clientId =
  "BONuo......";
const web3auth = new Web3Auth({
  clientId,
  chainConfig: {
    // this is ethereum chain config, change if other chain(Solana, Polygon)
    chainNamespace: CHAIN_NAMESPACES.EIP155,
    chainId: "0x1",
    rpcTarget: "https://mainnet.infura.io/v3/......",
  },
});
onMounted(async () => {
  await web3auth.initModal();
  await web3auth.connect();
});
```

**Reply by chaitanyapotti** [reply]
it might be due to an issue with your metamask.
sometimes, the window doesn't popup.
you just need to unlock metamask screen first and then try again

**Reply by MizukiSonoko** [reply]
I see, but I already unlocked, and in same browse, your site ( https://demo-app.web3auth.io/ ) works fine...

**Reply by shahbaz17** [reply]
@MizukiSonoko is this still happening? Have you tried in a different browser?
